docs/getting-started.rst

```diff
@@ -22,6 +22,15 @@ To validate the installation, start a Python shell or a Jupyter notebook and typ
 
 .. code-block:: python
 
-    import gettsim
-
-    gettsim.test()
+    import pandas as pd
+    from gettsim.config import ROOT_DIR
+    from gettsim.tax_transfer import tax_transfer
+    from gettsim.policy_for_date import get_policies_for_date
+
+    tax_data = yaml.safe_load((ROOT_DIR / "data" / "param.yaml").read_text())
+    year_of_policy = 2015  # We cover 1985 to 2019
+    tax_policy = get_policies_for_date(tax_policy_data, year=year)
+    tax_policy_pensions = pd.read_excel(ROOT_DIR / "data" / "pensions.xlsx").set_index(
```

Comment:
What is this? Why separate from the rest?

Reply:
This is the data for calculating the pensions(meanwages etc.) @Eric-Sommer always kept this separate because we are note sure yet, if this will be exogenously given or maybe estimated data. Maybe @Eric-Sommer you could elaborate on this!

Reply:
Ah, I see. See below. For now, we should target the use case that I want to have taxes and transfers for existing data. That means using the actual past values. In any case, we want to hide all of this from the user by default.